src/test/java/de/tum/in/www1/artemis/TextAssessmentIntegrationTest.java

```diff
@@ -112,6 +119,70 @@ public void testPrepareSubmissionForAssessment() {
         assertThat(result).isPresent();
     }
 
+    @Test
+    @WithMockUser(value = "instructor1", roles = "TA")
```

Comment:
not very important thing but its not consistent. 
```suggestion
    @WithMockUser(value = "tutor1", roles = "TA")
```